cmd/ftx/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/cmd/ftx"
+)
+
+// We could use this command to test FTX functions.
+func main() {
```

Comment:
how about:

    bbgo balance --session ftx

Reply:
Good idea. This PR is ready for another review, thanks